src/Config/Validator/Deploy/ResourceConfiguration.php

```diff
@@ -45,17 +54,26 @@ public function __construct(
      */
     public function validate(): Validator\ResultInterface
     {
+        $dbConfig = $this->dbConfig->get();
+        $resourceConfig = $this->resourceConfig->get();
         $wrongResources = [];
-        foreach ($this->resourceConfig->get() as $resourceName => $resourceData) {
-            if (!isset($resourceData['connection'])) {
+        foreach ($resourceConfig as $resourceName => $resourceData) {
+            if (!isset($resourceData[ResourceConfig::KEY_CONNECTION])
+                || !isset($dbConfig[DbConfig::KEY_CONNECTION][$resourceData[ResourceConfig::KEY_CONNECTION]])) {
```

Comment:
Please close multiline statements on the new line. For better code readiness.

Reply:
Done